developer-docs-site/docs/nodes/node-health-checker.md

```diff
@@ -206,8 +206,8 @@ There are more options available for which ports to use. Pass `-h` to see more o
 To generate the OpenAPI specs, run the following commands:
 
 ```
-cargo run -- server generate-openapi -f yaml > openapi.yaml
-cargo run -- server generate-openapi -f json > openapi.json
+cargo run -- server generate-openapi -f yaml > doc/spec.yaml
```

Comment:
Is this not supposed to be /spec/spec.yaml

Reply:
Nope, the API has spec files in `doc/` as well, I'm just matching that.